test/jest/widgets/common/BlueprintActions.test.ts

```diff
@@ -10,11 +11,18 @@ jest.mock('widgets/common/utils/PollHelper');
 const wait = jest.fn(() => Promise.resolve());
 const resetAttempts = jest.fn();
 
+const getRequiredBlueprintActionToolboxPart = (doGet: () => void) => {
+    return {
+        getManager: constant({ doPut: constant(Promise.resolve()), doGet })
+    };
```

Comment:
```suggestion
    } as unknown as WidgetlessToolbox;
```
and you should be able to avoid casting like that in L42 and L69.

Reply:
Oh yeah, nice tip!

Reply:
Applied within 66ab8c82bd0b8996b3bba665f4fe5cc0f469878c